common/v2/features/Dashboard/components/RecentTransactionList.tsx

```diff
@@ -200,15 +200,21 @@ export default function RecentTransactionList({ accountsList, className = '' }:
   const filteredGroups = groups.filter(group => group.entries.length !== 0);
 
   const recentTransactionsTable = {
-    head: ['Date', 'From Address', 'To Address', 'Amount', 'View More'],
+    head: [
+      translateRaw('RECENT_TRANSACTIONS_DATE') || 'Date',
```

Comment:
Why do we need the ` || 'Date'` here?

Reply:
https://github.com/MyCryptoHQ/MyCrypto/pull/3171#discussion_r417842123